Scoring period 3 (#21)
Joystream/founding-members: scoring-periods/3.md

## `Scoring Period 3`
### _15/03/21 - 28/03/21_
<br>

#### Announcement

> We are happy to announce the third "official" Scoring Period ([Scoring Period #0](/scoring-periods/0.md) was also created in order to recognize existing contributors to the project).

Reviewer: ```suggestion
```

> After the overwhelming amount of submissions for [Scoring Period #1](/scoring-periods/1.md), it was a lot more time consuming than anticipated to grade it, causing some grading delays and errors on our end.

Reviewer: ```suggestion
Due to the overwhelming number of submissions received for [Scoring Period #1](/scoring-periods/1.md), it ended up being significantly more time consuming than we had previously anticipated to grade, causing some delays and issues on our end.
```

> We will fix, and update these in the coming days, and start grading the submissions for [Scoring Period #2](/scoring-periods/2.md) in the the 17th of March.

Reviewer: ```suggestion
Needless to say, we plan to start grading the submissions for [Scoring Period #2](/scoring-periods/2.md) over the next week, and we hope we will be able to publish the results for this round much earlier now that we have learned some important lessons from the first round.
```

Again, here are some pointers to submitting based on what we learned during grading of Scoring Periods 0 and 1!

1. There is no need to submit your summary before the end of a Scoring Period!
  - It appears lots of people believe that it's a "race", and that you have to submit early and often.

>   - A summary should be the report of all your activities during the entire scoring period. In other words, unless you _know_ you will not make any further contributions, wait until the Scoring Period has ended.

Reviewer: ```suggestion
  - A summary should be the report of all your activities during the entire Scoring Period. In other words, unless you _know_ you will not make any further contributions, wait until the Scoring Period has ended.
  - There is a short "grace period" of about 4-5 days after the Scoring Period has ended where you can submit your summary, so as long as you don't wait too long too write your report, you should be fine!
```

  - Starting from this Scoring Period (ie. 3), we will reserve the right to only grade one summary per person per Scoring Period.

>     - As this our fault, we will of course not "punish" anyone for this for periods 1 or 2

Reviewer: ```suggestion
    - As the rules on this were not initially clear, we will of course not penalize anyone for having done this for periods 1 or 2
```

2. The program is not ending anytime soon!

>   - From interactions with the community, we have seen some people be of the impression that this is the one and only time to act in order to get points.

Reviewer: ```suggestion
  - From interactions with the community, we have seen some people be of the impression that there is a great rush to accumulate points.
```

  - In fact, we plan to keep the project running until we are close to mainnet launch which is least 9 months away.
  - If these plans change, we will notify the community in advance.
3. We _do_ have a referral program!
  - This has now been highlighted [here](/README.md#referral-program)
  - Refer your friends, and start earning points when they submit their summaries!
4. We have now made it so that just running a node will give (almost) as many points!

>   - Due to the competitiveness for the Validator spots making it hard for new users to get a spot, we needed to allow users to earn points simply for running (a) node(s).

Reviewer: ```suggestion
  - Due to competitiveness for the Validator spots making it hard for new users to get a spot, we felt that it was very important to allow new users (without a substantial tJOY balance) to earn points simply for running nodes, as this requires almost the same level of effort as running a Validator.
```
